express-notes-api/index.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+const fs = require('fs');
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const notesArray = [];
+  for (const property in data.notes) {
+    notesArray.push(data.notes[property]);
+  }
+  res.send(notesArray);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  if (req.params.id <= 0) {
```

Comment:
You need to also verify that the `id` is an integer. So you want to do something like:
```JAVASCRIPT
const id = Number(req.params.id);
if (!Number.isInteger(id) || id < 1) {
```